internal/search/result/owner.go

```diff
@@ -2,13 +2,45 @@ package result
 
 import (
 	"github.com/sourcegraph/sourcegraph/internal/api"
-	"github.com/sourcegraph/sourcegraph/internal/own/codeowners"
 	"github.com/sourcegraph/sourcegraph/internal/search/filter"
 	"github.com/sourcegraph/sourcegraph/internal/types"
 )
 
+type OwnerMatchOwner interface {
```

Comment:
```suggestion
type OwnerMatch interface {
```
? or were you trying to convey something else?

Reply:
There's already the result type `OwnerMatch` in here - this is the Owner field under the OwnerMatch so.. the OwnerMatchOwner? 😆 I didn't find a better name, but this one is taken unfortunately 😞

Reply:
ah I see. I mean we're tying these structs to search directly now, but would it make sense to just have
```go
type Owner interface { ... }
type OwnerPerson struct { ... }
type OwnerTeam struct { ... }

type OwnerMatch struct {
    ResolvedOwner Owner
}
```
if this is referenced from outside the pkg it would still be `search.Owner` so gives some distinction. wdyt?

Reply:
Like it!